.config/s3.config.php

```diff
@@ -0,0 +1,20 @@
+<?php
+if (getenv('OBJECTSTORE_S3_HOST')) {
+  $CONFIG = array (
+    'objectstore' => array(
+      'class' => '\\OC\\Files\\ObjectStore\\S3',
+      'arguments' => array(
+        'bucket' => getenv('OBJECTSTORE_S3_BUCKET'),
+        'autocreate' => getenv('OBJECTSTORE_S3_AUTOCREATE') ?: true,
+        'key'    => getenv('OBJECTSTORE_S3_KEY'),
+        'secret' => getenv('OBJECTSTORE_S3_SECRET'),
+        'hostname' => getenv('OBJECTSTORE_S3_HOST'),
+        'port' => getenv('OBJECTSTORE_S3_PORT'),
+        'use_ssl' => getenv('OBJECTSTORE_S3_SSL') ?: true,
+        'region' => getenv('OBJECTSTORE_S3_REGION'),
+        // required for some non Amazon S3 implementations
+        'use_path_style' => getenv('OBJECTSTORE_S3_USEPATH_STYLE') ?: true,
```

Comment:
I tried something like this and it didn't work with B2 S3 compatibility. Empty/blank doesn't respond well here. I had to have something like this:

`'use_path_style' => getenv('S3_USE_PATH_STYLE') === 'true' ?: false`

But I've also seen this:

`'use_path_style' => (bool) getenv('S3_USE_PATH_STYLE') ?: true`

I don't know if the latter works (haven't tried it).